### ISSUE TITLE
SFTP plugin not working

### ISSUE THREAD

**Issue by olejon** [afollestad/cabinet-issue-tracker#314]
I have installed the plugin and have the latest Cabinet Beta, and the plugin says `Connection successful` when trying it, but when connecting, it says `ClassNotFoundException when unmarshalling: com.afollestad.cabinet.plugins.PluginFile`

Nexus 5, running Android 6 (may that be the problem?)

**Comment by linuxman94** [comment]
Confirming the same issue here on my Nexus 5 with marshmallow

**Comment by afollestad** [comment]
Yeah, I can confirm it on my Nexus 5X but I haven't had enough time to fix it lately.

I or someone else will fix it soon.

**Comment by diegohdk** [comment]
Same here with Moto X 2014 on Lollipop.
